### ISSUE TITLE
New feature: Support multiple series bar chart in combined xy chart

### ISSUE THREAD

**Issue by GoogleCodeExporter** [adrianchia/achartengine#378]
```
What steps will reproduce the problem?
Define 3 series (2 bar chart series, 1 line chart series) in a dataset for a 
combined XY chart like this:
ChartFactory.getCombinedXYChartView(ctx, dataset, renderer, new String[] { 
BarChart.TYPE, BarChart.TYPE, LineChart.TYPE });


What is the expected output? What do you see instead?
Bar charts with multiple data series are not displayed properly in 
CombinedXYChart, the series overlap each other.

Current view: http://i.imgur.com/71HAPo3.jpg
Expected view: http://i.imgur.com/Jxq0BTV.jpg


What version of the product binary library are you using?
1.2.0

Please provide any additional information below.
The attached patch will enable the support of this feature.
Basically I extended the constructor with a more detailed chart type Definition 
which is used to properly build the data series and renderer for each chart.
I also extended the demo chart CombinedTemperatureChart to demonstrate the 
usage.



```

Original issue reported on code.google.com by `stefan.f...@gmail.com` on 11 Dec 2013 at 4:14

Attachments:
- [patch.txt](https://storage.googleapis.com/google-code-attachments/achartengine/issue-378/comment-0/patch.txt)
